src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -538,24 +537,18 @@ class AddItemsPage extends Component {
     return this.state.sortOrder;
   }
 
-  setLineItems(response, startIndex) {
+  setLineItems(response, startIndex, setAllFetchedItems) {
     const { data } = response.data;
-    let lineItemsData;
-
-    if (this.state.values.lineItems.length === 0 && !data.length) {
-      lineItemsData = new Array(1)
-        .fill({ sortOrder: 100, rowSaveStatus: RowSaveStatus.PENDING });
-    } else {
-      lineItemsData = _.map(data, val => ({ ...val, disabled: true }));
-    }
+    const lineItemsData = data.length ? _.map(data, val => ({ ...val, disabled: true })) :
+      new Array(1).fill({ sortOrder: 100, rowSaveStatus: RowSaveStatus.PENDING });
 
     const sortOrder = _.toInteger(_.last(lineItemsData).sortOrder) + 100;
     this.setState({
-      currentLineItems: this.props.isPaginated ?
+      currentLineItems: !setAllFetchedItems && this.props.isPaginated ?
```

Comment:
IIRC `startIndex == null` was indicating that it is not paginated. I am not sure if there is a need for `setAllFetchedItems`.

Reply:
Is it possible to use for it `startIndex == null`?